SQRLDotNetClientUI/ViewModels/MainMenuViewModel.cs

```diff
@@ -213,6 +230,95 @@ public void Login()
             }
         }
 
+        /// <summary>
+        /// Checks for new version in github and enables the Alert Button
+        /// </summary>
+        public void CheckForUpdates()
+        {
+
+            this.NewUpdateAvailable = GitAPIHubHelper.GitHubHelper.CheckForUpdates();
+        }
+
+
+        /// <summary>
+        /// Launches the installer to install a new update
+        /// </summary>
+        public async void InstallUpdate()
```

Comment:
I'm not sure if this should live in `MainMenuViewModel`. Should we put this in a more central/accessible place like a static general app class, so that it can be easily called from any view model? This way, we could start the auto-update right from the auth screen as well.

Reply:
yeah we should be able to move this pretty easily.